src/web/View.tsx

```diff
@@ -66,21 +67,33 @@ if (typeof document !== 'undefined') {
 
 export interface ViewContext {
     isRxParentAText?: boolean;
+    focusManager?: FocusManager;
 }
 
 export class View extends ViewBase<Types.ViewProps, {}> {
     static contextTypes: React.ValidationMap<any> = {
-        isRxParentAText: PropTypes.bool
+        isRxParentAText: PropTypes.bool,
+        focusManager: PropTypes.object
     };
     context: ViewContext;
 
     static childContextTypes: React.ValidationMap<any> = {
-        isRxParentAText: PropTypes.bool.isRequired
+        isRxParentAText: PropTypes.bool.isRequired,
+        focusManager: PropTypes.object
     };
 
+    private focusManager: FocusManager;
+
     private resizeDetectorAnimationFrame: number;
     private resizeDetectorNodes: { grow?: HTMLElement, shrink?: HTMLElement } = {};
 
+    constructor(props: Types.ViewProps) {
+        super(props);
+        if (this.props.restrictFocusWithin) {
```

Comment:
What happens if the "restrictFocusWithin" prop is updated? Looks like this case isn't handled. Probably requires adding a componentDidUpdate or componentWillUpdate method.